2-mandatory/2-oxygen-levels.js

```diff
@@ -1,17 +1,22 @@
-/*
-    Many years into the future, a team of Space Voyagers find their ship is low on Oxygen and need to dock
-    somewhere safe while they call home for help.
 
-    Their computer detects a list of nearby planets that have Oxygen in their atmosphere.
+    // Many years into the future, a team of Space Voyagers find their ship is low on Oxygen and need to dock
+    // somewhere safe while they call home for help.
 
-    To be safe, they need to land on the first unnamed planet that has Oxygen levels between 19.5% and 23.5%.
+    // Their computer detects a list of nearby planets that have Oxygen in their atmosphere.
 
-    Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
+    // To be safe, they need to land on the first unnamed planet that has Oxygen levels between 19.5% and 23.5%.
 
-    Some string methods that might help you here are .replace() and .substring().
-*/
+    // Write a function that finds the oxygen level of the first safe planet - Oxygen between 19.5% and 23.5%
 
-function findSafeOxygenLevel() {}
+    // Some string methods that might help you here are .replace() and .substring().
+function isSafeOxygenLevel(oxygenlevel){
+  return parseFloat(oxygenlevel)>19.5 && parseFloat(oxygenlevel)<23.5;
```

Comment:
This one is intersting! this is not passing the third test(thet you commented:)). To pass this test, you should add a condition or use includes()method to only include strings which have %.
Or forget about parseFloat and use .replace() and .substring() and .map() obviously